packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts

```diff
@@ -24,3 +24,37 @@ export function uploadFile({
   // stub
   return;
 }
+
+/**
+ * Format bytes as human-readable text.
+ *
+ * @param bytes Number of bytes.
+ * @param si True to use metric (SI) units, aka powers of 1000. False to use
+ *           binary (IEC), aka powers of 1024.
+ * @param dp Number of decimal places to display.
+ *
+ * @return Formatted string.
+ */
+export function humanFileSize(bytes, si = false, dp = 1) {
```

Comment:
This needs some unit tests (happy and unhappy).

Reply:
I'll add this in a separate PR